doc/classes/StringName.xml

```diff
@@ -35,6 +35,12 @@
 		</constructor>
 	</constructors>
 	<methods>
+		<method name="bbcode_escape" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the string with BBCode tags escaped, which makes them ineffective when used in [RichTextLabel] with [member bbcode_enabled] set to [code]true[/code]. This is useful for handling user input to prevent BBCode injection. See also [method strip_bbcode].
```

Comment:
I think it's worth mentioning that escaping is implemented for Godot BBCode. There is no single BBCode standard and other implementations may not support the `[lb]` tag.